packages/arb-token-bridge-ui/src/util/withdrawals/fetchWithdrawals.ts

```diff
@@ -143,27 +147,35 @@ export async function fetchWithdrawals({
     }
   }
 
-  if (isAlchemy) {
-    // for alchemy, fetch sequentially
-    queries.push({ receiver, gateways: [gateways.standardGateway] })
-    queries.push({ receiver, gateways: [gateways.wethGateway] })
-    queries.push({ receiver, gateways: [gateways.customGateway] })
-    queries.push({ receiver, gateways: gateways.otherGateways })
-  } else {
-    // for other chains, fetch in parallel
-    queries.push({ receiver, gateways: allGateways })
+  /// receiver queries; only add if nonce > 0 for orbit chains
+  const fetchReceivedTransactions =
+    isCoreChain || (isOrbitChain && senderNonce > 0)
```

Comment:
shouldn't this be the child chain nonce of the receiver address instead of `senderNonce`?

Reply:
it can't be, if we used that and received has no outgoing transactions, it wouldn't show ETH withdrawals in tx history